routes/index.js

```diff
@@ -3,8 +3,11 @@ const router = express.Router()
 const db = require('../db')
 const mailClient = require('../mail/index.js')
 const bodyParser = require('body-parser')
-router.use(bodyParser.json())
 const { v4: uuidv4 } = require('uuid')
+const que = require('./queries')
+const pro = require('./dataprocess.js')
+const { addUser } = require('./queries')
```

Comment:
Unless you're interacting with an Object-Oriented class-like structure where it makes sense to call functions on the module object (eg `db` --> `db.query`), use the object destructuring syntax `{ functionName, functionName2 } = require(...)` to obtain direct references to function names. 

That is, change line 8 to be like line 9, and 

merge line 7 with line 9.
OR 
remove ` const db = require('../db')` and do `const db = require('./queries')`

Reply:
Ahh that must have been leftover code from testing. `const que = require('./queries')` should take care of that. Good catch.